crates/ruff_linter/src/rules/flake8_implicit_str_concat/rules/explicit.rs

```diff
@@ -35,45 +39,133 @@ use crate::settings::LinterSettings;
 pub(crate) struct ExplicitStringConcatenation;
 
 impl Violation for ExplicitStringConcatenation {
```

Comment:
Nit: You can implement `AlwaysFixableViolation` instead of specifying `FIX_AVAILABILITY = Always